pkg/sql/pai_submitter.go

```diff
@@ -200,11 +202,14 @@ func (s *paiSubmitter) submitPAITask(code, paiCmd, requirements string) error {
 	if e != nil {
 		return e
 	}
+	cw := &logChanWriter{wr: s.Writer}
+	var output bytes.Buffer
+	w := io.MultiWriter(cw, &output)
+	defer cw.Close()
 	cmd := exec.Command("odpscmd", "--instance-priority", "9", "-u", cfg.AccessID, "-p", cfg.AccessKey, "--project", cfg.Project, "--endpoint", cfg.Endpoint, "-e", paiCmd)
-	if out, err := cmd.CombinedOutput(); err != nil {
-		return fmt.Errorf("failed %s, %s, %v", cmd, out, err)
-	}
-	return nil
+	cmd.Stdout, cmd.Stderr = w, w
+	fmt.Println(paiCmd)
```

Comment:
Try using `log.Printf` to avoid multi-thread output buffer flushing issue.

Reply:
Done.